frontend/src/pages/modelRegistry/screens/ModelVersionDetails/ModelVersionDetailsHeaderActions.tsx

```diff
@@ -32,75 +38,119 @@ const ModelVersionsDetailsHeaderActions: React.FC<ModelVersionsDetailsHeaderActi
           >
-            Actions
-          </MenuToggle>
+            Deploy
+          </Button>
+        </ActionListItem>
+        {isFineTuningEnabled && (
+          <ActionListItem
+            className="pf-v5-u-w-100"
+            data-testid="lab-tune-button"
+            aria-label="Lab tune version"
+          >
+            <Button variant="secondary" onClick={() => setIsLabTuneModalOpen(true)}>
+              LAB tune
+            </Button>
+          </ActionListItem>
         )}
-      >
-        <DropdownList>
-          <DropdownItem
-            isAriaDisabled={hasDeployment}
-            id="archive-version-button"
-            aria-label="Archive model version"
-            key="archive-version-button"
-            onClick={() => setIsArchiveModalOpen(true)}
-            tooltipProps={
-              hasDeployment ? { content: 'Deployed model versions cannot be archived' } : undefined
-            }
-            ref={tooltipRef}
+        <ActionListItem>
+          <Dropdown
+            isOpen={isOpenActionDropdown}
+            onSelect={() => setOpenActionDropdown(false)}
+            onOpenChange={(open) => setOpenActionDropdown(open)}
+            popperProps={{ position: 'right', appendTo: 'inline' }}
+            toggle={(toggleRef) => (
+              <MenuToggle
+                variant={ButtonVariant.secondary}
+                ref={toggleRef}
+                onClick={() => setOpenActionDropdown(!isOpenActionDropdown)}
+                isExpanded={isOpenActionDropdown}
+                aria-label="Model version details action toggle"
+                data-testid="model-version-details-action-button"
+              >
+                Actions
+              </MenuToggle>
+            )}
           >
-            Archive model version
-          </DropdownItem>
-        </DropdownList>
-      </Dropdown>
-      {isDeployModalOpen ? (
+            <DropdownList>
+              <DropdownItem
+                isAriaDisabled={hasDeployment}
+                id="archive-version-button"
+                aria-label="Archive model version"
+                key="archive-version-button"
+                onClick={() => setIsArchiveModalOpen(true)}
+                tooltipProps={
+                  hasDeployment
+                    ? { content: 'Deployed model versions cannot be archived' }
+                    : undefined
+                }
+                ref={tooltipRef}
+              >
+                Archive model version
+              </DropdownItem>
+            </DropdownList>
+          </Dropdown>
+        </ActionListItem>
+      </ActionListGroup>
+      {isLabTuneModalOpen ? (
+        <StartRunModal
+          onCancel={() => setIsLabTuneModalOpen(false)}
+          onSubmit={(selectedProject) => {
+            navigate(
+              `${getModelCustomizationPath(selectedProject)}?${new URLSearchParams(
+                tuningData!,
+              ).toString()}`,
```

Comment:
This isn't how we want to pass the params - you want to use router history state, not URL params.

```ts
navigate(`${getModelCustomizationPath(selectedProject)}`, { state: tuningData });
```

You can see if it worked correctly by making sure the params are visible in the ilab form. with your current version they show as `-`.